AzureTable/Microsoft.DataTransfer.AzureTable/IDefaults.cs

```diff
@@ -11,5 +11,10 @@ public interface IDefaults
         /// Gets the default value that indicates which internal Azure Table fields should be preserved in the source adapter output.
         /// </summary>
         AzureTableInternalFields SourceInternalFields { get; }
+
+        /// <summary>
+        /// Get the default value that indicated which location mode should be used 
```

Comment:
Minor styling issue:
- `indicated` -> `indicates`
- dot "`.`" at the end to be consistent with the rest